File: backend/auto-trading-bot/src/main/java/com/trading/autotradingbot/dto/AccountSummaryDto.java
package com.trading.autotradingbot.dto;

import lombok.Data;
import java.math.BigDecimal;

@Data
public class AccountSummaryDto {
    private BigDecimal currentBalance;
    private BigDecimal currentPortfolioValue;
    private BigDecimal initialCapital;
    private BigDecimal totalProfitLoss;
}

File: frontend/src/types/dto/AccountSummaryDto.ts
import type { BigDecimalString } from '../../common/types';

export interface AccountSummaryDto {
    currentBalance: BigDecimalString; // Current cash balance (USDT)
    currentPortfolioValue: BigDecimalString;
    initialCapital: BigDecimalString;
    totalProfitLoss: BigDecimalString;
}